Software/Development/SemesterProject/App/analogAcquisitionManager.c

```diff
@@ -346,8 +356,8 @@ void createAnalogSignals(void)
      * before using them.
      * Then pass the maximum and then the minimum value of the threshold you want to set.
      */
-    setCurrentThresholds(&currentThreshold[0], 250, -250);
-    setDCLinkVoltageThresholds(&DCLinkVoltageThreshold[0], 42, 25);
+    setCurrentThresholds(&currentThreshold[0], 275, -275);
+    setDCLinkVoltageThresholds(&DCLinkVoltageThreshold[0], 42, 0);
```

Comment:
Mybe now that we have feedback we could change this value back to something more realistic, this way if the supply limits current we will also go to error mode.

Reply:
This is a good argument. I have put the DC-link at minimum 5V.